pages/api/users/index.js

```diff
@@ -0,0 +1,51 @@
+import Iron from "@hapi/iron";
+import Cookies from "cookies";
+import dbConnect from "../../../utils/connectMongo";
+import User from "../../../models/User";
+
+dbConnect();
+export const ENC_KEY =
+  "qrshsqrwdzxmidxczavnlzxxgfyrompjewvieydljozlsetcsngdjbnrixeqcmpncepljuhxlhtobfzyareajmn";
+
+const loginInfo = async (req, res) => {
+  const { method } = req;
+  const { username, password } = req.body;
+  const infoUser = { username: username, password: password, name: "default" };
+
+  switch (method) {
+    case "GET":
+      try {
+        const users = await User.find();
+        res.status(200).json({ success: true, data: users });
+      } catch (error) {
+        res.status(405).json({ success: false });
+      }
+      break;
+
+    case "POST":
```

Comment:
A possible refactor here to use either switch/case or only if statements for control flow